src/js/themes/base.d.ts

```diff
@@ -512,6 +512,7 @@ export interface ThemeType {
     };
     color?: ColorType;
     extend?: ExtendType;
+    focusIndicator: boolean;
```

Comment:
I don't think we should force the user to use this prop.
```suggestion
    focusIndicator?: boolean;
```